startup.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+if [ -f /app/config.sh ]; then
+  if [ ! -f /app/.config_run ]; then
+    /app/config.sh
```

Comment:
I don't see this script. What's this for?

Reply:
I added this yesterday to install custom binaries into the docker container to use more command line tools to run...

You could add a config.sh in your mounted /app folder and run commands like:
```apt update
apt upgrade -y
apt install netcat```
to use nc (netcat) in your checks.. :) (The Base-Image python:2.7 does not ship with any additional command line tools)

Regards
SeaLife